feat: command line argument for specifying output path

charlesbdudley opened the pull request.

tlvince commented on cli.js:
```diff
@@ -32,14 +33,26 @@ const download = (url, cb) => {
+  },
+]
+const args = commandLineArgs(argumentDefinitions)
+const outputPath = args.outputPath || './coverage/badge.svg'
```
Looks like you could use commandLineArgs's [defaultValue](https://github.com/75lb/command-line-args/blob/9c07457557351d3db825bd5a3d17c053d6a10dea/doc/option-definition.md#module_option-definition--OptionDefinition+defaultValue) instead.

charlesbdudley replied:
thanks @tlvince When you get a chance can you please review the changes?

charlesbdudley commented:
@tlvince Ah, didn't understand what the "standard" meant in the test script. Fixed now.

Also, I added the `defaultOption` key in the command-line-args config to set this up for potential future command line options. Let me know if you'd rather remove the flag completely.

shawnpetros commented:
hey, is this getting merged soon? I really wanted to use this and am looking for this exact functionality!

tlvince commented:
:tada: This PR is included in version 1.1.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/make-coverage-badge)
- [GitHub release](https://github.com/tlvince/make-coverage-badge/releases/tag/v1.1.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
